packages/atoms/src/ListItem/SelectableListItem.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { TouchableOpacity, TouchableOpacityProps, View } from 'react-native';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { ListItemIcon } from './ListItemIcon';
+import { ListItemText } from './ListItemText';
+import { Radio, RadioProps } from '../Radio';
+import { BaseListItem } from './listItemTypes';
+import { Box } from '../Box';
+
+type SelectableListItemProps = RadioProps &
+    Omit<BaseListItem, 'onPress'> &
+    Omit<TouchableOpacityProps, 'style' | 'onPress'>;
+
+const radioStyle = prepareNativeStyle(() => ({
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'flex-end',
+}));
+
+export const SelectableListItem = ({
+    iconType,
+    title,
+    subtitle,
+    style,
+    value,
+    onPress,
+    isTextWrapped = false,
+    isChecked = false,
+    isDisabled = false,
+    ...props
+}: SelectableListItemProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <TouchableOpacity
```

Comment:
what happen when you press this ListItem? I think it should behave same like if you press Radio directly. Is it way how it works now?

Reply:
Fixed in https://github.com/trezor/trezor-suite/pull/5482/commits/81e57f45bd044390d8b19895426a9516cff9256b